index.html

```diff
@@ -0,0 +1,255 @@
+          Mobilizing more financial support for developing countries, and Making the pivot toward implementation.
+        </p>
+        </div>
+        <h2 class="date">6 Nov - 20 Nov 2022</h2>
+        <p class="address">
+          @ UN Climate Change
+        </p>
+      </section>
+
+      <!-- main-program -->
+      <section id="program" class="main-program">
+        <h2 class="program-title">Main Program</h2>
+        <hr class="line1" />
+        <div class="program">
+          <div class="program-tab">
+            <img src="./img/program_icon_01.png" alt="icon" />
+            <h3 class="program-text1">Lecture</h3>
+            <p class="program-text2">
+              Lorem, ipsum dolor sit amet consectetur adipisicing elit.
+              Consectetur harum velit.
+            </p>
+          </div>
+          <div class="program-tab">
+            <img src="./img/program_icon_02.png" alt="icon" />
+            <h3 class="program-text1">CC Exhibition</h3>
+            <p class="program-text2">
+              Lorem, ipsum amet consectetur adipisicing elit. Consectetur harum
+              velit.
+            </p>
+          </div>
+          <div class="program-tab">
+            <img src="./img/program_icon_03.png" alt="icon" />
+            <h3 class="program-text1">Forum</h3>
+            <p class="program-text2">
+              Lorem, ipsum dolor sit amet consectetur adipisicing elit.
+              Consectetur harum velit.
+            </p>
+          </div>
+          <div class="program-tab">
+            <img src="./img/program_icon_04.png" alt="icon" />
+            <h3 class="program-text1">Workshop</h3>
+            <p class="program-text2">
+              Lorem, ipsum dolor sit amet consectetur adipisicing elit.
+              Consectetur harum velit.
+            </p>
+          </div>
+          <div class="program-tab">
+            <img src="./img/program_icon_05.png" alt="icon" />
+            <h3 class="program-text1">CC Ignite</h3>
+            <p class="program-text2">
+              Lorem, ipsum dolor sit amet consectetur adipisicing elit.
+              Consectetur harum.
+            </p>
+          </div>
+          <a href="https://unfccc.int/cop27#attend" class="join-btn" target="_blank">Join Climate Change Conference</a>
+        </div>
+      </section>
+
+      <!-- Speakers -->
+      <section class="speakers">
+        <h2>featured Speakers</h2>
+        <hr class="line1" />
+        <div class="speakersContainer"></div>
+        <button class="btn show_more">
+          <a href="#">MORE</a>
+          <img src="./img/arrow_down.png" alt="arrow icon" />
+        </button>
+        <a href="#" class="btn show_less">LESS</a>
+      </section>
+
+      <!-- partners -->
+      <section id="partner" class="partner desk-ver">
+        <h2 class="program-title">Partner</h2>
+        <hr class="line1" />
```

Comment:
[REQUIRED]The speakers' images should have a chess-like design behind them just like the template. You are not also using the text colours present in the template. 
![image](https://user-images.githubusercontent.com/62938735/212387092-a77e2233-952a-4a5b-9f74-d7c687e17903.png)